perl_modules/Atlas/ZoomaClient.pm

```diff
@@ -122,20 +123,20 @@ more ontology URIs.
 
 sub map_term {
 
-    my ( $self, $propertyType, $propertyValue ) = @_;
+    my ( $self, $propertyType, $propertyValue, $organism ) = @_;
```

Comment:
If we are changing the method's signature, shouldn't a call for map_term be modified elsewhere? Is this not impacting our previous usage of the ZoomaClient now that we are adding organism there? Have we tested it in both bulk and SC usage scenarios?

Reply:
The change @suhaibMo made to the condense SDRF script in https://github.com/ebi-gene-expression-group/atlas-prod/pull/119 is still present and part of the experiment_metatdata repo, see https://github.com/ebi-gene-expression-group/experiment_metadata/blob/1afc9fd63cb224f09ce74d48423b8fcdc0f1cb06/condense_sdrf.pl#L639.

This PR is not proposing new changes, it's reinstating code that should not have been lost.